services/app-web/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -70,7 +71,10 @@ const StatusTag = ({
 export const Dashboard = (): React.ReactElement => {
     const { loginStatus, loggedInUser } = useAuth()
     const location = useLocation()
-    const { testFrontendBanner } = useFlags()
+
+    // Get the LD feature flag
+    const ldClient = useLDClient()
+    const testFrontendBanner = ldClient?.variation('test-frontend-banner')
```

Comment:
Hmmm, how can we have some type control over flags? Should the current flag names should be hard coded constants in a file?

Maybe the flags could be stored as a constant in common-code so both api and web have the same source of truth for the flags ist. When we add a new flag to the codebase, which requires a code change anyway, we add it to the list 

Want to make sure someone can't typo this randomly in the code.

Reply:
Yeah I like this idea, it'd force us to keep a running list of flags somewhere, and it could be helpful for us when we get to the point where we have flags we want to remove.

I'll give it a shot and push it up to this PR. Thanks!

Reply:
Nice! Also on the topic of typescript check out this `getFlagValue` wrapper. 

Let's create a wrapper like this also instead of directly calling ldClient in the React code. Usually this makes things easier to refactor and improve this code as we move along. Even if all our version of `getFlagValue` does is declare the type of the flags as `LaunchDarkly.LDFlagValue` instead of `any` at this points (since I know we are initializing the user a bit differently).
https://launchdarkly.com/blog/using-launchdarkly-with-typescript/

Reply:
So I was writing a wrapper like this, but it turns out that [`useLDClient`](https://docs.launchdarkly.com/sdk/client-side/react/react-web#hooks) is a React hook and needs to be called in the component. I started writing something where we'd have to pass the client in to the wrapper function, but since I added the constants file where we pass a known safe flag to `ldClient.variation()` it didn't feel like it was saving us much. 

You think it's still worth it? Happy to add it if so.